### ISSUE TITLE
Error when using Revise with DataFrames

### ISSUE THREAD

**Issue by nalimilan** [timholy/Revise.jl#56]
I consistently get this error when trying to use Revise with DataFrames on Julia 0.6.
```
julia> using Revise

julia> using DataFrames
INFO: Recompiling stale cache file /home/milan/.julia/lib/v0.6/DataFrames.ji for module DataFrames.

# EDIT file src/dataframerow/utils.jl

julia> 1
Revise is currently tracking the following files: String["/home/milan/.julia/StatsBase/src/misc.jl", "/home/milan/.julia/CategoricalArrays/src/CategoricalArrays.jl", "/home/milan/.julia/StatsBase/src/hist.jl", "/home/milan/.julia/StatsBase/src/weights.jl", "/home/milan/.julia/SpecialFunctions/src/SpecialFunctions.jl", "/home/milan/.julia/StatsBase/src/common.jl", "/home/milan/.julia/Revise/src/Revise.jl", "/home/milan/.julia/WeakRefStrings/src/WeakRefStrings.jl", "/home/milan/.julia/SpecialFunctions/src/deprecated.jl", "/home/milan/.julia/StatsBase/src/deprecates.jl", "/home/milan/.julia/StatsBase/src/cov.jl", "/home/milan/.julia/StatsBase/src/robust.jl", "/home/milan/.julia/CategoricalArrays/src/pool.jl", "/home/milan/.julia/SortingAlgorithms/src/SortingAlgorithms.jl", "/home/milan/.julia/CategoricalArrays/src/deprecated.jl", "/home/milan/.julia/DataFrames/src/DataFrames.jl", "/home/milan/.julia/CategoricalArrays/src/recode.jl", "/home/milan/.julia/StatsBase/src/moments.jl", "/home/milan/.julia/StatsBase/src/counts.jl", "/home/milan/.julia/StatsBase/src/sampling.jl", "/home/milan/.julia/DataStreams/src/DataStreams.jl", "/home/milan/.julia/StatsBase/src/ranking.jl", "/home/milan/.julia/StatsBase/src/deviation.jl", "/home/milan/.julia/SpecialFunctions/src/bessel.jl", "/home/milan/.julia/CategoricalArrays/src/array.jl", "/home/milan/.julia/CategoricalArrays/src/nullablearray.jl", "/home/milan/.julia/StatsBase/src/rankcorr.jl", "/home/milan/.julia/SpecialFunctions/src/erf.jl", "/home/milan/.julia/CategoricalArrays/src/subarray.jl", "/home/milan/.julia/StatsBase/src/signalcorr.jl", "/home/milan/.julia/StatsBase/src/toeplitzsolvers.jl", "/home/milan/.julia/Nulls/src/Nulls.jl", "/home/milan/.julia/CategoricalArrays/src/typedefs.jl", "/home/milan/.julia/SpecialFunctions/src/gamma.jl", "/home/milan/.julia/StatsBase/src/statmodels.jl", "/home/milan/.julia/Reexport/src/Reexport.jl", "/home/milan/.julia/StatsBase/src/StatsBase.jl", "/home/milan/.julia/CategoricalArrays/src/value.jl", "/home/milan/.julia/StatsBase/src/scalarstats.jl", "/home/milan/.julia/StatsBase/src/empirical.jl", "/home/milan/.julia/CategoricalArrays/src/extras.jl", "/home/milan/.julia/CategoricalArrays/src/buildfields.jl"]
ERROR (unhandled task failure): /home/milan/.julia/DataFrames/src/dataframerow/utils.jl is not currently being tracked.
Stacktrace:
 [1] revise_file_now(::String) at /home/milan/.julia/Revise/src/Revise.jl:609
 [2] revise() at /home/milan/.julia/Revise/src/Revise.jl:633
 [3] macro expansion at /home/milan/.julia/Revise/src/Revise.jl:772 [inlined]
 [4] (::Revise.##17#18{Base.REPL.REPLBackend})() at ./event.jl:73
```

**Comment by timholy** [comment]
Revise has to be able to figure out which files are `include`d by parsing the source, and there are limits to how smart it is. Does DataFrames use loops or something?

If that's it, it will be fixed by #49 (but it requires 0.7).

**Comment by nalimilan** [comment]
Of course, I could have figured it out on my own. That's [the most absurd loop](https://github.com/JuliaData/DataFrames.jl/blob/86c61452634dd96dbe420705ceb69410e86986b5/src/DataFrames.jl#L74) I can think of, so I could just remove it. :-)
